india_compliance/gst_india/client_scripts/purchase_invoice.js

```diff
@@ -9,9 +9,11 @@ frappe.ui.form.on(DOCTYPE, {
 
     after_save(frm) {
         if (
-            frm.doc.docstatus ||
             frm.doc.supplier_address ||
-            !is_e_waybill_applicable(frm)
+            !frm.doc.gst_category === "Unregistered" ||
```

Comment:
use !==
Not logical. Should always check for applicability and threshold if unregistered or is_return. Check similarly in Sales Invoice